src/haddock/modules/design/optint/defaults.yaml

```diff
@@ -0,0 +1,128 @@
+random_seed:
+  default: 42
+  type: integer
+  min: 0
+  max: 999
+  title: Random seed
+  short: Random seed
+  long: Random seed
+  group: ""
+  explevel: guru
+
+number_of_generations:
+  default: 25
+  type: integer
+  min: 1
+  max: 150
+  title: Number of generations
+  short: The number of generations to run the optimizer for
+  long: The number of generations to run the optimizer for
+  group: ""
+  explevel: guru
+
+population_size:
+  default: 250
+  type: integer
+  min: 10
+  max: 1000
+  title: Population size
+  short: The number of individuals in each generation
+  long: The number of individuals in each generation
+  group: ""
+  explevel: guru
+
+cxpb:
+  default: 0.6
+  type: float
+  min: 0
+  max: 1
+  precision: 2
+  title: Crossover probability
+  short: The probability of crossover
+  long: The probability of crossover
+  group: ""
+  explevel: guru
+
+mutpb:
+  default: 0.8
+  type: float
+  min: 0
+  max: 1
+  precision: 2
+  title: Mutation probability
+  short: The probability of mutation
+  long: The probability of mutation
+  group: ""
+  explevel: guru
+
+eta:
+  default: 0.8
+  type: float
+  min: 0
+  max: 1
+  precision: 2
+  title: Mutation strength
+  short: The strength of the mutation
+  long: The strength of the mutation
+  group: ""
+  explevel: guru
+
+indpb:
+  default: 0.2
+  type: float
+  min: 0
+  max: 1
+  precision: 2
+  title: Independent probability
+  short: The probability of independent mutation
+  long: The probability of independent mutation
+  group: ""
+  explevel: guru
+
+output_models:
+  default: 1
+  type: integer
+  min: 1
+  max: 150
```

Comment:
Why 150? I guess the max must be related to the population of the genetic algorithm, correct?

Reply:
It is yes, could be any number actually, but also something we should test - will increase for now

Reply:
I guess in principle only the best models will be written - so probably a small number anyway